src/zzz_od/gui/view/devtools/icon_editor_dialog.py

```
@@ -225,8 +235,8 @@ def _on_delete_clicked(self):
             icon = self.current_icon_list[self.selected_row]
             reply = QMessageBox.question(
                 self,
-                '确认删除',
-                f'确定要删除图标 "{icon.icon_name}" ({icon.template_id}) 吗？\n此操作不可撤销。',
+                gt('确认删除'),
+                f'{gt("确定要删除图标")} "{icon.icon_name}" ({icon.template_id}) {gt("吗？")}\n{gt("此操作不可撤销。")}',
```

Ruff (0.15.13)

[warning] String contains ambiguous `？` (FULLWIDTH QUESTION MARK). Did you mean `?` (QUESTION MARK)?

(RUF001)